sanic/app.py

```diff
@@ -443,6 +443,11 @@ def dispatch(
         inline: bool = False,
         reverse: bool = False,
     ) -> Coroutine[Any, Any, Awaitable[Any]]:
+        if self.config.get("SKIP_TOUCHUP", False):
```

Comment:
If we are setting it as a proper config value, then we do not need to use `dict` syntax to get the value.